### ISSUE TITLE
Class not found exception

### ISSUE THREAD

**Issue by tdalianis** [HEADS-project/ApamaComponents#3]
I am preparing a test project based on the apama components, but when I run the project (mvn kev:run) I get class not found exception.
 
The problem seems to be related to the external libraries, since the exception is thrown when the ApamaSubscriber tries to inject the monitor script engineClient.injectMonitorScript(epl); 

FATAL : EngineClientBean.connect() - caught Error (not Exception), will re-throw
com/ctc/wstx/stax/WstxInputFactory
java.lang.NoClassDefFoundError: com/ctc/wstx/stax/WstxInputFactory
        at com.apama.correlator.corba.CorrelatorManagement_TCP.<init>(Correlator
Management_TCP.java:250)
        at com.apama.engine.EngineManagementFactory.connectToEngineViaTCP(Engine
ManagementFactory.java:157)
        at com.apama.engine.EngineManagementFactory.connectToEngineViaTCP(Engine
ManagementFactory.java:120)
        at com.apama.engine.EngineManagementFactory.connectToEngine(EngineManage
mentFactory.java:56)
        at com.apama.engine.beans.EngineClientBean.connect(EngineClientBean.java
:1318)
        at com.apama.engine.beans.EngineClientBean.getEngineManagement(EngineCli
entBean.java:285)
        at com.apama.engine.beans.EngineClientBean.injectMonitorScript(EngineCli
entBean.java:2573)
        at com.apama.engine.beans.EngineClientBean.injectMonitorScript(EngineCli
entBean.java:2564)
        at atc.test.cep.TestApamaSubscriber.start(TestApamaSubscriber.java:99)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.
java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAcces
sorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.kevoree.library.java.wrapper.KInstanceWrapper.startInstance(KInst
anceWrapper.java:105)
        at org.kevoree.library.java.wrapper.ComponentWrapper.startInstance(Compo
nentWrapper.java:54)
        at org.kevoree.library.java.command.StartStopInstance.run(StartStopInsta
nce.java:36)

**Comment by maxleiko** [comment]
I got rid of the issue by re-installing the local SNAPSHOT of the JavaNode. This might be due to the fact that we are working with SNAPSHOT since a long time now and some got lost at some point? I don't know. Or maybe you have a JavaNode which is outdated in your own Maven repo. I advise you, if it's the case, to remove all kevoree-related artifacts from any custom Maven repos you have.

TL;DR: `rm -rf ~/.m2/repository/org/kevoree/library/java/org.kevoree.library.java.javaNode/` did the trick

**Comment by skorsky** [comment]
For an Apama Java client (the ApamaComponents are such a client), you need all jars in the class path. The Class-path directives in the Jar Manifests tell which jars to use:

engine_client5.3.jar needs the util5.3.jar. (Class-path: util5.3.jar)
And the util5.3.jar tells us:
Class-path: jdom.1.0.jar log4j-1.2.15.jar looks-2.0.4.jar forms-1.0.7.
 jar xstream-1.4.3.jar xpp3_min-1.1.4c.jar xmlpull-1.1.3.1.jar stax-ap
 i-1.0.1.jar wstx-asl-3.2.4.jar

The wstx jar contains the classes mentioned in the stack trace above.

**Comment by tdalianis** [comment]
Indeed the issue is resolved.

Thank you
